src/Adapter/PlatformServices.Desktop/Utilities/FileUtility.cs

```diff
@@ -293,9 +293,15 @@ internal virtual string[] GetDirectoriesInADirectory(string directoryPath)
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes", Justification = "Requirement is to handle all kinds of user exceptions and message appropriately.")]
         private string GetSymbolsFileName(string path)
         {
+            Debug.Assert(!string.IsNullOrEmpty(path), "path");
+
             try
             {
-                return DiaHelper.GetSymbolsFileName(path);
+                string pdbFile = Path.ChangeExtension(path, ".pdb");
+                if (File.Exists(pdbFile))
+                {
+                    return pdbFile;
+                }
             }
             catch (Exception ex)
```

Comment:
When InvalidChars are there in File Path will throw exception. Checked locally it was cached in catch block